src/app.js

```diff
@@ -31,7 +32,21 @@ io.of('/game').on('connection', (socket) => {
   socket.on(MOVE_PLAYED, async ({ gameId, playerId, move }) => {
     const game = await dynamo.getGameById(gameId)
     const { piece, position } = move
+
+    if (isInvalidMove(game, playerId, position)) {
+      console.log('Invalid move!')
```

Comment:
It's a good idea when doing logging to add additional information. Something like:
`console.log('Player ${pId} tried an illegal move in game ${}')`
This would make the logs a lot more useful